colossalai/fx/passes/algorithms/linearize.py

```diff
@@ -2,11 +2,12 @@
 from torch.fx import GraphModule, Node
 
 
-def linearize(gm: GraphModule) -> List[List[Node]]:
+def linearize(gm: GraphModule, cnode: List[str] = None) -> List[List[Node]]:
```

Comment:
where do you get the cnode?

Reply:
yes, it should be added to solver

Reply:
I mean how do you get the list of cnode?

Reply:
The first list of common nodes should be input by users, and the other common nodes will be propagated in the following codes.